common/src/main/java/com/dfsek/terra/config/pack/ConfigPack.java

```diff
@@ -340,4 +328,8 @@ public SamplerCache getSamplerCache() {
     public CheckedRegistry<TerraStructure> getStructureRegistry() {
         return new CheckedRegistry<>(structureRegistry);
     }
+
+    public WorldConfig toWorldConfig(TerraWorld world){
+        return new WorldConfig(world, this, main);
```

Comment:
Might be best if this value is lazily loaded instead of reinstanciated each time.

Reply:
This method is only invoked upon `TerraWorld` instantiation, which only happens once per world per reload. We *could* lazily load it, but it would be redundant to do so, as it would never actually be used.

Reply:
I still think that it should be lazily loaded. Because theoretically in the future, we might want addons to (for some reason) get a world config value.

Reply:
In that case they can get the WorldConfig instance from the TerraWorld, which is what everything else does. Ideally we lock down the `toWorldConfigs` so addons *can't* touch it.

Reply:
`WorldConfig#getConfig` allows getting the `WorldConfig` instance attached to a world.

Reply:
fair